### ISSUE TITLE
"Unhandled rejection Error: 404 Not Found" during artifact publish

### ISSUE THREAD

**Issue by Lange** [electron-userland/electron-builder#325]
Hey there,

My [Linux](https://travis-ci.org/nodecg/dashboard/jobs/123626030), [OSX](https://travis-ci.org/nodecg/dashboard/jobs/123626031), and [Windows](https://ci.appveyor.com/project/Lange/dashboard) builds all fail with a `404` error when attempting to publish artifacts. My [`.travis.yml`](https://github.com/nodecg/dashboard/blob/master/.travis.yml) and [`appveyor.yml`](https://github.com/nodecg/dashboard/blob/master/appveyor.yml) files have my `GH_TOKEN`, and the token has the `repo_deployment` and `public_repo` scopes.

This is my first application with electron-builder, so forgive me if I have overlooked something.

Thanks!

**Comment by develar** [comment]
Works for me (e.g. https://github.com/develar/onshape-desktop-shell/releases).

Please provide full console output.

**Comment by Lange** [comment]
I linked to my CI builds in my opening comment, you should be able to see the logs there.

**Comment by develar** [comment]
Because your repo belongs not to user, but to org (I suppose). it is not yet supported — our bug. Will be fixed today/tomorrow. Thanks for project/CI links.

**Comment by Lange** [comment]
Ahh okay, thanks for looking into it!

**Comment by develar** [comment]
@Lange No, it seems we cannot blame GitHub API — please double check that your github token is correct. 

GitHub have no choice — GitHub can report 404 (not found) instead of 403 (forbidden) to not expose file status (yeah, security, security, security).

I have added test to check — I got 404 if token is not correct. And works if token is valid.

**Comment by Lange** [comment]
Hm. I have generated a new token and given it all `repo` permissions. My builds are still failing in the same manner:

- [Windows](https://ci.appveyor.com/project/Lange/dashboard/build/1.0.35)
- [Linux](https://travis-ci.org/nodecg/dashboard/jobs/123913576)
- [OSX](https://travis-ci.org/nodecg/dashboard/jobs/123913577)
- [.travis.yml](https://github.com/nodecg/dashboard/blob/master/.travis.yml)
- [appveyor.yml](https://github.com/nodecg/dashboard/blob/master/appveyor.yml)

I generated my token via the "Personal Access Tokens" on my settings page. I don't believe there is a way for me to generate a token for the organization itself, so I think what I'm doing is correct. I'm not sure what I'm doing wrong.

**Comment by develar** [comment]
[new test](https://github.com/develar/electron-builder/commit/f7bc61bf13eba23a9d04e6c47a3fba5b9418c0e7#diff-f5a6da74c1a2cdf92d6b93f9eaa0daccR62) passed on CI.

I will add debug log and publish new version to understand what's wrong.

It is not easy to debug since GitHub doesn't report real status :(

**Comment by develar** [comment]
Please set version to `next` and try again (error message will be a bit clear)

**Comment by Lange** [comment]
Well, the problem was an rather silly oversight on my part.

When I first started writing this app, its repo was `nodecg/nodecg-dashboard`. I changed it to be just `nodecg/dashboard`, but I forgot to change the `name` and `git.url` in `package.json`. As a result, the deploy script was attempting to deploy to `https://api.github.com/repos/nodecg/nodecg-dashboard/releases` instead of `https://api.github.com/repos/nodecg/dashboard/releases`.

Thank you so much for taking the time to help me troubleshoot this issue. My deployments are working now.
